samples/Core-Bot/datatypes_timex_expression/date_range.py

```diff
@@ -0,0 +1,7 @@
+
+class DateRange:
+
+    def __init__(self, start, end):
```

Comment:
>start [](start = 23, length = 5)

Random q: Are these assumed to be datetimes? 
(We've been sort of following [https://www.python.org/dev/peps/pep-0526/](https://www.python.org/dev/peps/pep-0526/).  Highly possible this is not "pythonic" enough, but we should probably be consistent.)